nextjsUIapp/src/app/data-release/page.tsx

```diff
@@ -0,0 +1,85 @@
+
+};
+
+export default async function Contact() {
+    const current_session = await getServerSession(authOptions);
+    //user is logged in so redirect to admin page
+    if (current_session) return redirect("/admin");
+    return (
+        <div className="set-margin-hundred">
+            <h2 className="mb-4 text-3xl font-extrabold font-extrabold leading-none text-sky-900 animate-slide-up">Data
+                Release</h2>
+
+
+            <p className="mb-3 font-normal text-justify font-light text-sky-900 animate-slide-up">
+                BrainKB serves as a knowledge base platform that provides scientists worldwide with tools for
+                searching, exploring, and visualizing Neuroscience knowledge represented by knowledge graphs (KGs).
+                Moreover, BrainKB provides cutting-edge tools that enable scientists to contribute new information
+                (or knowledge) to the platform and is expected to be a go-to destination for all
+                neuroscience-related research needs.
+
+            </p><br/><br/>
+
+
+            <div className="relative overflow-x-auto shadow-md sm:rounded-lg animate-slide-up">
+                <table className="w-full text-sm text-left rtl:text-right text-gray-500 dark:text-gray-400">
+                    <thead className="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+                    <tr>
+                        <th scope="col" className="px-6 py-3">
+                            Date
+                        </th>
+                        <th scope="col" className="px-6 py-3">
+                            <div className="flex items-center">
+                                Release
+
+                            </div>
+                        </th>
+                        <th scope="col" className="px-6 py-3">
+                            <div className="flex items-center">
+                               Individual KGs by Type
+
+                            </div>
+                        </th>
+                        <th scope="col" className="px-6 py-3">
+                            <div className="flex items-center">
+                                Combined
+                            </div>
+                        </th>
+
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr className="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                        <th scope="row"
+                            className="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white">
+                            NA
+                        </th>
+                        <td className="px-6 py-4">
+                            Na
+                        </td>
+                        <td className="px-6 py-4">
+                            NA
+                        </td>
+                        <td className="px-6 py-4">
+                            NA
```

Comment:
what do these `NA` values mean?

Reply:
This is a skeleton page. Will be updated later.